api/yelp_beans/send_email.py

```diff
@@ -183,6 +183,14 @@ def create_google_calendar_invitation_link(user_list, title, office, location, m
         'location': office + " " + location,
         'add': ','.join([user.email for user in user_list])
     }
+    try:
+        if meeting_datetime.strftime("%Z") and "ctz" not in url_params:
```

Comment:
We construct the `url_params` above and it looks like we don't set the `ctz` param. Will the second part of this if statement ever be false?

Also is `strftime("%Z")` being truthy to check if the datetime has a timezone or not? If so could `meeting_datetime.tzinfo is not None` instead to make it clearer what this is checking?

Reply:
good catch, refactored